src/Compilers/CSharp/Portable/FlowAnalysis/DefiniteAssignment.cs

```diff
@@ -846,15 +846,17 @@ private void NoteRead(BoundNode fieldOrEventAccess)
             }
         }
 
-        protected virtual void NoteWrite(Symbol variable, BoundExpression value, bool read)
+        protected virtual void NoteWrite(Symbol variable, BoundExpression value, bool read, bool isRef)
```

Comment:
Does `isRef` mean literally `ref` or any kind like `in / out`. Consider adding a comment here to make the intent clear.

Reply:
It means "ref assignment", so there's no in/out variant. Added comment.